### ISSUE TITLE
Test failing with TreeTime 0.11.2 release

### ISSUE THREAD

**Issue by victorlin** [nextstrain/augur#1392]
[translate-with-gff-and-locus-tag.t](https://github.com/nextstrain/augur/blob/9cd915a2a8a2d63244fc7f7d0316b51f5c3ef936/tests/functional/translate/cram/translate-with-gff-and-locus-tag.t) is [failing](https://github.com/nextstrain/augur/actions/runs/7615222099/job/20739237313):

```
--- tests/functional/translate/cram/translate-with-gff-and-locus-tag.t
+++ tests/functional/translate/cram/translate-with-gff-and-locus-tag.t.err
@@ -15,[12](https://github.com/nextstrain/augur/actions/runs/7615222099/job/20739237313#step:9:13) +[15](https://github.com/nextstrain/augur/actions/runs/7615222099/job/20739237313#step:9:16),21 @@
   >   --output-node-data aa_muts.json \
   >   --alignment-output translations.vcf \
   >   --vcf-reference-output translations_reference.fasta
-  Gene length of 'rrs' is not a multiple of 3. will pad with N
+  ERROR: The VCF file '/home/runner/work/augur/augur/tests/functional/translate/cram/../data/tb/nt_muts.vcf' had genotype calls of ploidy 2 but sample 'NODE_0000001' has a genotype of ploidy 1
+  
+  
+  ERROR from TreeTime: This error is most likely due to a problem with your input data.
+  Please check your input data and try again. If you continue to have problems, please open a new issue including
+  the original command and the error above:  <https://github.com/nextstrain/augur/issues/new/choose>
+  
   Read in 187 specified genes to translate.
   Read in 188 features from reference sequence file
-  [16](https://github.com/nextstrain/augur/actions/runs/7615222099/job/20739237313#step:9:17)2 genes had no mutations and so have been be excluded.
-  amino acid mutations written to .* (re)
+  [2]
```

@jameshadfield can you check if the test file is bad and needs to be updated?

**Comment by jameshadfield** [comment]
Yes. It was changed in https://github.com/nextstrain/augur/pull/1355/commits/2c9db828b588ff2fb2cb582aabe0c3bac3a2cde1 (PR #1355). Do you want to help coordinate release of that PR with me?

P.S. I expected us to tightly pin treetime, given it's aversion to major releases -- any particular reason we aren't?

**Comment by victorlin** [comment]
Also, we're coordinating Augur release 24.0.0 [on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1705954949739629).

**Comment by victorlin** [comment]
Addressed in #1355.
